Civility Optics/Views/VenueDetails.swift

```diff
@@ -76,7 +76,24 @@ var email: String
         ForEach(model.results, id: \.self) { result in
           HStack {
             VStack {
+                Group{
+                HStack(spacing: 4) {
```

Comment:
We must optimize this as a refactor ticket in the next sprint. We need create components otherwise it times out in the build.